### ISSUE TITLE
[Typo] Inconsistent/excessive dashes on p. 63

### ISSUE THREAD

**Issue by V2Blast** [AshleyMoni/Lancer-Community-Edition#275]
**The Problem:**
The paragraph below the mission goals table uses a lot of em dashes, but one of them appears incorrectly as a pair of hyphens:

"Success of a mission depends on completing the mission goal, but missions can end without the goal(s) being completed — indeed, some missions end so poorly that they leave players — or their factions **--** worse off than before they embarked. Characters that end a mission — successful or not — always gain 1 license level."

In addition, the number of em dashes here is a little excessive, making it a little hard to parse the sentence.

Finally, em dashes traditionally should not be followed or preceded by spaces; if spacing is desired, en dashes should be used instead.

**The Solution:**
Most straightforwardly, change the pair of hyphens after "factions" to an em dash. Even if no other change is made, this should be done. Likewise, if em dashes are used, spaces before and after them should be removed (otherwise they should be changed to en dashes).

Then, it may be worth considering which of these might be worth eliminating or changing to other punctuation. In this case, the easiest way to reduce the number of dashes without changing the meaning or deemphasizing something that should be emphasized would probably be to change the dashes around "or their factions" to parentheses instead:

"Success of a mission depends on completing the mission goal, but missions can end without the goal(s) being completed—indeed, some missions end so poorly that they leave players (or their factions) worse off than before they embarked. Characters that end a mission—successful or not—always gain 1 license level."

**The Location:**
p. 63, below the "Mission Goals" table

**Comment by melodycwatson** [comment]
Fixed! :)
